README.md

```diff
@@ -278,11 +279,39 @@ Bootstrap JS are loaded.
 
 > **Note**: In general, this is a bad idea.
 
+### Images
+
+Don't forget to remove images that are no longer used.  Try to keep the images
+sorted in the local `images/` directory, with names that naturally group related
+images together in alphabetical order; for instance prefer
+`settings-file-share.png` and `settings-proxies.png` to
+`file-share-settings.png` and `proxies-settings.png`.
+
+On Mac, when applicable capture windows rather than capture rectangular regions;
+this eliminates unpleasant background appearing in the documentation, and saves
+the editors the need to crop.  Capture windows without shadows.  To this end,
+once you pressed `Command-Shift-4`, press Option while clicking on the window.
+To disable shadows once for all, run
```

Comment:
Since you're helping us a lot with docs let me give you a bit of feedback about styleguide so that we are consistent across the board.

* Whenever you think about using `;` just use `.`.
* End the sentence before a snippet with `:`. In this case `To disable ..., run:`

Reply:
Thanks for these tips, and please, do continue teaching me :)